Source/Core/CommandLineOptions.cs

```diff
@@ -1786,6 +1786,11 @@ identity function.
      {:subsumption n}
        Overrides the /subsumption command-line setting for this assertion.
 
+     {:focus}
+      It deletes all paths that don't have the containing block, assumes code leading to the block,
```

Comment:
Documentation for other attributes begin at the column right after the :.

Splits verification into two problems. First problem deletes all paths that do not have the focus block. Second problem considers the paths deleted in the first problem and does not contain either the focus block or any block dominated by it.